timemanager-webapp/src/views/Contracts.vue

```diff
@@ -0,0 +1,21 @@
+<script setup>
+import ButtonComponent from "@/components/general/ButtonComponent.vue";
+import { VPdfViewer } from "@vue-pdf-viewer/viewer";
+import { ref } from "vue";
+const showPdf = ref(false);
+const pdfUrl = new URL('../asset/contract/Contrat de travail.pdf', import.meta.url).href;
+const togglePdf = ()=>{
+    showPdf.value = !showPdf.value;
+}
+</script>
+<template>
+    <div>
+        <ButtonComponent title="Contracts" @click="togglePdf">
+        </ButtonComponent>
```

Comment:
<ButtonComponent title="Contracts" @click="togglePdf"/>

better to write it like that